ref-tests/tests/ic-ref.rs

```diff
@@ -1378,6 +1380,20 @@ mod extras {
             let result = ic00.canister_status(&canister_id).call_and_wait().await?;
             assert_eq!(result.0.settings.log_visibility, LogVisibility::Public);
 
+            // TODO: re-enable when supported.
```

Comment:
since the test is running against the replica version shipped with dfx and allowed viewers is not available there yet I remove this commented test for now. should be added after replica version gets updated.